src/gfx/opt.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#if defined(__clang__) || defined(__GNUC__)
+#define likely(x) __builtin_expect((x), 1)
```

Comment:
hmm we have those in chainblocks.h

Reply:
Copied because I wanted to keep the dependency graph `gfx --> chainblocks-extra` It's no longer used now so I removed it.